backend/.idea/backend.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should add the `.idea` folder within the `.gitignore` file because those files are IDE dependant and they shouldn't be pushed on the remote repository